gene_finder.py

```diff
@@ -30,42 +33,89 @@ def get_complement(nucleotide):
     >>> get_complement('C')
     'G'
     """
-    # TODO: implement this
-    pass
+    if nucleotide == 'A':
+        return 'T'
+    elif nucleotide == 'T':
+        return 'A'
+    elif nucleotide == 'C':
+        return 'G'
+    elif nucleotide == 'G':
+        return 'C'
+    else:
+        return 'This is not a nucleotide'
```

Comment:
Another way to hand errors :+1: 
`raise ValueError(nucleotide, " is not a nucleotide")`